game/src/common/warp.rs

```diff
@@ -173,7 +173,7 @@ fn warp_to_id(ctx: &mut EventCtx, app: &mut App, line: &str) -> Option<Transitio
         Ok(idx) => match line.chars().next().unwrap() {
             'r' => {
                 let r = app.primary.map.maybe_get_r(RoadID(idx))?;
-                ID::Lane(r.children_forwards[0].0)
```

Comment:
I believe this was always true before by default, but actually there _was_ a bug -- if you made an edit and reversed the only forward lane of a road, this would have broken. :D